course3/homework-list/benchmark.cc

```diff
@@ -0,0 +1,24 @@
+#include "list.h"
+
+#include <list>
+#include <forward_list>
+#include <algorithm>
+#include <benchmark/benchmark.h>
+
+
+template<template<typename...> class ListType>
+void ListBenchmark(benchmark::State& state)
+{
+	ListType<int> l;
+	for (int i=0; i < state.range_x(); ++i)
+		l.push_front(i);
+}
+
+//BENCHMARK_TEMPLATE(ListBenchmark, Karun::List)->Range(1<<0, 1<<10);
```

Comment:
BENCHMARK_TEMPLATE and BENCHMARK_MAIN don't compile. I get an "Undefined symbols for architecture x86_64" error. I'm running the code on OS X with clang++.